CRM/Moregreetings/Upgrader.php

```diff
@@ -43,4 +43,21 @@ public function upgrade_5001() {
     return TRUE;
   }
 
+
+  /**
+   * Make sure users get a warning about the switch to APIv4
+   *
+   * @return bool TRUE on success
+   * @throws Exception
+   */
+  public function upgrade_5002() {
+    CRM_Core_Session::setStatus(
+      E::ts('MoreGreetings has switched to APIv4, which means that some tokens might not work any more.') . '<br/><br/>' .
+           E::ts('An example would be the defunct <code>$individual_prefix</code> token, which can be substituted by the <code>prefix_id:label</code>') . '<br/><br/>'.
```

Comment:
This would be `contact.individual_prefix` token (without the `$`), no …?

Reply:
I haven't configured this in a while, but I think its SMARTY tokens, not regular ones - see [the examples](https://docs.civicrm.org/moregreetings/en/latest/example/)